frontend/src/components/Card.vue

```diff
@@ -7,7 +7,7 @@
 <style scoped>
 .card {
   position: relative;
-  margin: 10px 0px;
```

Comment:
This affects multiple components are you sure this is non breaking?

Reply:
Yeah I mean I've tested it everywhere card is used.